pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -83,7 +83,7 @@ class ModelCheckpoint(Callback):
             Please note that the monitors are checked every `period` epochs.
             if ``save_top_k >= 2`` and the callback is called multiple
             times inside an epoch, the name of the saved file will be
-            appended with a version count starting with `v0`.
+            appended with a version count starting with `v1`.
```

Comment:
```suggestion
            appended with a version count starting with ``v1``.
```

Reply:
also since you added `STARTING_VERSION`, maybe update this a little bit?

Reply:
Updated it! Have a look:

Reply:
nice!